docs/admin/logging/journald.md

```diff
@@ -42,6 +42,9 @@ You can set the logging driver for a specific container by using the
 Users can use the `--log-opt NAME=VALUE` flag to specify additional
 journald logging driver options.
 
+Users can use the `--log-opt journal-gateway-address=host:port` option to
+specify the `systemd-journal-gatewayd` address where the driver connects to.
+
 ### tag
```

Comment:
Yes, I have a plan to send an additional pull request to support `systemd-journal-remote`.